.storybook/preview-head.html

```diff
@@ -64,10 +64,21 @@
     border-top-left-radius: 4px;
   }
 
+  .doctable tr:last-child td:first-child, .doctable tr:last-child th:first-child {
+    border-bottom-left-radius: 4px;
+  }
+
+  .doctable tr:last-child td:last-child, .doctable tr:last-child th:last-child {
+    border-bottom-right-radius: 4px;
+  }
+
+  .doctable tr:first-child td:last-child, .doctable tr:first-child th:last-child {
+    border-top-right-radius: 4px;
+  }
+
   .doctable td {
```

Comment:
very minor, maybe for a follow up.

What do you think about putting a light border between table rows?

Reply:
should be there in Chrome. Are you looking at this in Safari? This is really the exact same table as the normal prop table... the rows may not be there in Safari.